src/Features/Core/Portable/MakeMethodSynchronous/AbstractMakeMethodSynchronousCodeFixProvider.cs

```diff
@@ -77,7 +77,7 @@ private async Task<Solution> RenameThenRemoveAsyncTokenAsync(Document document,
             var syntaxPath = new SyntaxPath(node);
 
             // Rename the method to remove the 'Async' suffix, then remove the 'async' keyword.
-            var newSolution = await Renamer.RenameSymbolAsync(solution, methodSymbol, newName, solution.Options, cancellationToken).ConfigureAwait(false);
+            var newSolution = await Renamer.RenameSymbolAsync(solution, methodSymbol, new SymbolRenameOptions(), newName, cancellationToken).ConfigureAwait(false);
```

Comment:
should all these cases be SymbolRenameOptions.Default?

Reply:
In this case I don't think there is a benefit of having Default static since it's the same as `default(SymbolRenameOptions)` or `new SymbolRenameOptions()`. The record ctor has all optional parameters and they all default to false.